src/Modules/StatCan.OrchardCore.SaaSConfiguration/Views/Admin/ExecuteRecipe.cshtml

```diff
@@ -0,0 +1,50 @@
+@model StatCan.OrchardCore.SaaSConfiguration.ViewModels.ExecuteRecipeViewModel
+<zone Name="Title"><h1>@RenderTitleSegments(T["Execute Recipe for Tenants"])</h1></zone>
+
+<style asp-name="codemirror"></style>
+<script asp-name="codemirror" depends-on="admin" at="Foot"></script>
+<script asp-name="codemirror-addon-display-autorefresh" at="Foot"></script>
+<script asp-name="codemirror-mode-javascript" at="Foot"></script>
+
+@Html.ValidationSummary()
+
+<form asp-action="Json">
+
+  <div class="form-group" asp-validation-class-for="SelectedTenantNames">
+    <label asp-for="SelectedTenantNames">@T["Tenants"]</label>
+    <div class="row">
+      @for (var i = 0; i < Model.AllTenants.Count; i++)
```

Comment:
Create a Razor component for this as it might be useful for many different views